package.json

```diff
@@ -4,13 +4,18 @@
   "description": "",
   "main": "index.js",
   "scripts": {
-    "test": "echo \"Error: no test specified\" && exit 1"
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "dev": "nodemon index.js"
   },
   "author": "",
   "license": "ISC",
   "dependencies": {
     "axios": "^1.3.3",
     "cheerio": "^1.0.0-rc.12",
-    "discord.js": "^14.7.1"
+    "discord.js": "^14.7.1",
```

Comment:
maybe we should use fixed package versions to avoid some package changes. you can add a `.npmrc` with `save-exact=true` to ensure next libraries

also, where is `package-lock` file?